tools/README.md

```diff
@@ -0,0 +1,21 @@
+# CLI tool uri2pem.py
+
+Simple tool to create PEM files for PKCS#11 URI
+Usage:
+
+    python uri2pem.py --help
+    python uri2pem.py 'pkcs11:token=MyToken;object=MyObject;type=private'
+    python uri2pem.py --bypass 'someBogusURI'
+    # output
+    python uri2pem.py --out mykey.pem 'pkcs11:token=MyToken;object=MyObject;type=private'
+    # verification, if token is available, requires --out <filename>
+    python uri2pem.py --verify --out mykey.pem 'pkcs11:token=MyToken;object=MyObject;type=private'
+
+The tool doesn't validate the argument for a valid PKCS#11 URI
+
+## Tests
+
+If you run `make check` the tool has a test suite that will run
+against NSS softoken in `../tests/tmp.softokn/tests`.
```

Comment:
This comment says that running make check tests the tool ... but I see no changes to the Make files that would make this true.

Did you just miss that in the commits?

Reply:
Updated README.md to state - first run `make check` in order to create the testing NSS softoken, then in `tools/` run `pytest tests`.